configs/sandpack-contents/component-theming/skeleton.js

```diff
@@ -0,0 +1,117 @@
+module.exports = {
+  App: `import {
+    Text,
+    Box,
+    Flex,
+    Skeleton,
+    Button,
+    IconButton,
+    useColorMode,
+  } from "@chakra-ui/react";
+  import { FaMoon, FaSun } from "react-icons/fa";
+  import { useState } from "react";
+  
+  export default function App() {
+    const [loading, setLoading] = useState(true);
+    const { toggleColorMode, colorMode } = useColorMode();
+  
+    return (
+      <Box pos="relative">
+        <Flex textAlign="center" gap={3} mt={3} direction="column">
+          <Box>
+            <Text>Default Skeleton</Text>
+            <Skeleton isLoaded={!loading} bg="orange" fadeDuration={1}>
+              <Text>Hidden Text</Text>
+            </Skeleton>
+          </Box>
+  
+          <Box>
+            <Text>With xl size</Text>
+            <Skeleton isLoaded={!loading} bg="orange" fadeDuration={1} size="xl">
+              <Text>Hidden Text</Text>
+            </Skeleton>
+          </Box>
+  
+          <Box>
+            <Text>With red variant</Text>
+            <Skeleton
+              isLoaded={!loading}
+              bg="orange"
+              fadeDuration={1}
+              variant="red"
+            >
+              <Text>Hidden Text</Text>
+            </Skeleton>
+          </Box>
+  
+          <Box>
+            <Button colorScheme={"blue"} onClick={() => setLoading(!loading)}>
+              Toggle
+            </Button>
```

Comment:
Let's name this:
Toggle loading state

And if you can restyle the button since it doesn't go well with the skeletons.